providers/direct/src/test/java/quarks/test/providers/direct/DirectJobTest.java

```diff
@@ -260,4 +296,17 @@ protected static void sleep(long millis) {
             }
         }
     }
+    
+    private static class Configuration {
```

Comment:
What's the value of this class, seems to be just a JsonObject?

I know it's a test, but people are likely to copy & paste from tests for their applications so I'm not sure this is a pattern that should be encouraged.